src/go/rpk/pkg/cli/cmd/root.go

```diff
@@ -18,13 +18,11 @@ import (
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/common"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
 	"golang.org/x/crypto/ssh/terminal"
 )
 
-const feedbackMsg = `We'd love to hear about your experience with redpanda:
```

Comment:
As far as I can see feedbackMsg is used only in start command. I can not find another place where we could print this message.

Reply:
Also here: https://github.com/vectorizedio/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/root.go#L84

Reply:
Thanks!